extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -0,0 +1,209 @@
+	private createMigrationContainer!: azdata.FlexContainer;
+	private _view!: azdata.ModelView;
+
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.IR_PAGE_TITLE), migrationStateModel);
+	}
+
+	protected async registerContent(view: azdata.ModelView): Promise<void> {
+		this._view = view;
+
+		const createNewController = view.modelBuilder.button().withProps({
+			label: constants.NEW,
+			width: '100px',
+		}).component();
+
+		createNewController.onDidClick((e) => {
+			this.createMigrationContainer.display = 'inline';
+		});
+
+		const setupButtonGroup = 'setupOptions';
+
+		this.defaultSetupRadioButton = view.modelBuilder.radioButton().withProps({
+			label: constants.DEFAULT_SETUP_BUTTON,
+			name: setupButtonGroup
+		}).component();
+		this.defaultSetupRadioButton.checked = true;
+
+		this.customSetupRadioButton = view.modelBuilder.radioButton().withProps({
+			label: constants.CUSTOM_SETUP_BUTTON,
+			name: setupButtonGroup
+		}).component();
+
+		this.startSetupButton = view.modelBuilder.button().withProps({
+			label: 'Create',
+			width: '100px'
+		}).component();
+
+		this.startSetupButton.onDidClick((e) => {
+			if (this.defaultSetupRadioButton.checked) {
+				vscode.window.showInformationMessage(constants.FEATURE_NOT_AVAILABLE);
+			} else {
+				this.createMigrationContainer.display = 'none';
+				const dialog = new CreateIntegrationRuntimeDialog(this.migrationStateModel, this);
+				dialog.initialize();
+			}
+		});
+
+		this.cancelSetupButton = view.modelBuilder.button().withProps({
+			label: 'Cancel',
+			width: '100px'
+		}).component();
+
+		this.cancelSetupButton.onDidClick((e) => {
+			this.createMigrationContainer.display = 'none';
+		});
+
+		const setupButtonsContainer = view.modelBuilder.flexContainer().withItems([
+			this.startSetupButton,
+			this.cancelSetupButton
+		],
+			{ CSSStyles: { 'margin': '10px', } }
+		).withLayout({
+			flexFlow: 'row'
+		}).component();
+
+		this.createMigrationContainer = view.modelBuilder.flexContainer().withItems(
+			[
+				this.defaultSetupRadioButton,
+				this.customSetupRadioButton,
+				setupButtonsContainer
+			]
+		).withLayout({
+			flexFlow: 'column'
+		}).component();
+
+		this._connectionStatus = view.modelBuilder.text().component();
+
+		this.createMigrationContainer.display = 'none';
+
+		const form = view.modelBuilder.formContainer()
+			.withFormItems(
+				[
+					{
+						component: this.migrationControllerDropdownsContainer()
+					},
+					{
+						component: createNewController
+					},
+					{
+						component: this.createMigrationContainer
+					},
+					{
+						component: this._connectionStatus
+					}
+
+				]
+			);
+		await view.initializeModel(form.component());
+	}
+
+	public async onPageEnter(): Promise<void> {
+		this.populateMigrationController();
+		this.wizard.registerNavigationValidator((pageChangeInfo) => {
```

Comment:
Same here - why aren't you using the normal page validation logic to determine whether the dropdown is valid?